pylint/checkers/variables.py

```diff
@@ -846,6 +846,7 @@ def __init__(self, linter=None):
         self._checking_mod_attr = None
         self._loop_variables = []
         self._type_annotation_names = []
+        self._except_handler_names = []
```

Comment:
```suggestion
        self._except_handler_names = []
        """This is a queue, last in first out"""
```

Reply:
Can we do something with this information in the attribute name? `_except_handler_names_queue`? (Haven't had my coffee, name inspiration is low 😄 )

Reply:
`_except_handler_names_lifo` ?